modules/seqexec/server/src/main/scala/seqexec/server/niri/NiriControllerSim.scala

```diff
@@ -29,7 +30,7 @@ object NiriControllerSim extends NiriController {
   override def observeProgress(total: Time): fs2.Stream[IO, Progress] =
     sim.observeCountdown(total, ElapsedTime(0.seconds))
 
-  override def calcTotalExposureTime(cfg: DCConfig): Time = {
+  override def calcTotalExposureTime(cfg: DCConfig): IO[Time] = IO.pure {
```

Comment:
Why are some `calcTotalExposureTime` methods parameterized with `F[Time]`, and others use `IO[Time]`?

Reply:
Because I'm still trying to convert parts from `IO` to `F`. A full change is next to impossible so in places I keep `IO`